src/diff/index.js

```diff
@@ -288,15 +288,15 @@ export function applyRef(ref, value, ancestorComponent) {
  * @param {import('../internal').Component} ancestorComponent The parent
  * component to this virtual node
  */
-export function unmount(vnode, ancestorComponent) {
+export function unmount(vnode, ancestorComponent, skipRemove) {
```

Comment:
So `skipRemove` here is a flag indicating that the mounted DOM node has already been removed because its parent was removed? It would be useful to have a JSDoc comment to that effect.